shared/lib_battery_voltage.h

```diff
@@ -36,8 +36,11 @@ struct voltage_params {
     };      // voltage model (0), voltage table (1)
 
     MODE voltage_choice;
-    int num_cells_series;        // number of cells in series
-    int num_strings;             // addition number in parallel
+    // int overflow erroeous results
+//    int num_cells_series;        // number of cells in series
```

Comment:
Please remove commented out code (leaving the bit about the int causing an overflow is fine)